ibis/backends/base/sql/alchemy/translator.py

```diff
@@ -69,6 +74,9 @@ def integer_to_timestamp(self, arg, tz: str | None = None):
 
     supports_unnest_in_select = True
 
+    get_sqla_type = staticmethod(dtype_to_sqlalchemy)
```

Comment:
Perhaps we should figure out a nicer way to inject the converter functions to the backend classes, any idea?